cerberus/aws_auth.py

```diff
@@ -59,6 +63,6 @@ def get_token(self):
 
         token = resp.json()['client_token']
         iam_principal_arn = resp.json()['metadata']['aws_iam_principal_arn']
-        print('Successful authenticated with Cerberus as {}'.format(iam_principal_arn))
```

Comment:
The motivation to change this to use logging is that I'm working on a cli client, and wanted to be able to output data as JSON in scripts.  I realized that these print statements were being included in my output to stdout.

Reply:
Can you log it to are err and accomplish what you want?

Reply:
Can you log it to are err and accomplish what you want?

Reply:
That's the point.  Is that instead of print statements, the library is now set up for logging.  And is following the advice for other python libraries in that you plumb it to log ,and then leave it up to the client to activate logging in whatever way they see fit.